packages/owl-bot/src/owl-bot.ts

```diff
@@ -104,7 +104,7 @@ export = (privateKey: string | undefined, app: Probot, db?: Db) => {
       // Fetch the .Owlbot.lock.yaml from the head ref:
       const lock = await core.getOwlBotLock(
         head.repo.full_name,
-        context.payload.number,
+        context.payload.pull_request.number,
```

Comment:
`context.payload.number` was an alias for `context.payload.pull_request.number`, I've moved us to consistently using `context.payload.pull_request.number` (since using both was confusing in tests).